app/views/homerooms/show.html.erb

```diff
@@ -74,19 +74,19 @@
             <th class="risk sort-default" data-sort-method="reverse_number">
               <span class="table-header">Risk</span>
             </th>
-            <th class="program">
+            <th class="program" data-sort-method="prog_assign_sort">
```

Comment:
@ronny2205 Check out my [comments on the issue](https://github.com/studentinsights/studentinsights/issues/155#issuecomment-199256848) about this.  While this works I think it's hard to discover what's going on here.  We should do something to help clarify what this is, even if it's just the suggestion to add a comment about what's going on.